### ISSUE TITLE
Iron Ore Textures Difficult to See

### ISSUE THREAD

**Issue by Dark-Arcana** [HexBugOrion/UnEarthed#76]
Mod version: 1.2.1

There are 4 Unearthed stone textures for which iron ore is nearly impossible to see. These stone blocks are: siltstone, sandstone, mudstone, and beige limestone. For accessibility reasons (poor vision, color blindness, etc.), the iron ore textures should be modified in some way to provide more clear contrast.

These 4 blocks also do not have the iron ore texture on the top or bottom of the block. That makes it even harder to find the ores in the world. These blocks should be modified to add the iron ore texture to all block faces.

Here is an image to show all blocks side by side (also featured is an ice block):

https://cdn.discordapp.com/attachments/874474961086087209/879454156795019274/2021-08-23_12.55.56.png
